test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -110,14 +111,21 @@ def inline_flatten_list(inputs, res_list):
 
 def run_ort(ort_sess, input):
     input = flatten_tuples(input)
-    input = to_numpy(input)
+    input_final = []
+    for input_ in input:
+        input_final.extend(unpack_quantized_tensor(input_))
+    input = to_numpy(input_final)
```

Comment:
Naming the previous thing input_final, and then not using it as the final input is quite confusing.
I'd name it `input_unpacked`.